apps/src/javalab/JavalabEditorTabMenu.jsx

```diff
@@ -48,14 +50,49 @@ class JavalabTabMenuComponent extends Component {
         >
           Delete
         </button>
-        {showVisibilityOption && (
+        {showVisibilityOption && !fileIsVisible && (
           <button
             type="button"
             key="visibility"
```

Comment:
Can we change this key now to match the others? like "starter" or something?

Reply:
ahh yes!